### ISSUE TITLE
Please clearify plugin dependencies.

### ISSUE THREAD

**Issue by AndKe** [OllisGit/OctoPrint-PrintJobHistory#44]
A new user is greeted with word "dependencies" which is kind of the opposite of "optional":
```
To get the real power of the Print Job History - Plugin you need the following plugins installed/enabled:

PreHeat (missing)
FilamentManager (missing)
DisplayLayerProgress (missing)
UltimakerFormatPackage (missing)
```
Please describe a little about what use they provide... what kind of statistics or what a user will be missing out by not having them. And, can any of those be added, producing extra data, at a later time.

UltimakerFormatPackage is maybe just used for preview if no camera is attached?

FilamentManager page says "This plugin has been abandoned"  - and it makes more sense to have filaments in the slicing software, since their properties affect slicing of linear advance setting.

It is hard to imagine what information from DisplayLayerProgress is useful in history.

Preheat: ther is no plugin with that name in the standard repository.

**Comment by Schokobecher** [comment]
FilamentManager is also not Python3 compatible since it's abandoned so it would make sense to ditch as many "dependencies" or "optional" things as possible.

**Comment by OllisGit** [comment]
Hi,
I changed the text for more clarification in the newest release 1.1.0.

**Comment by leoheck** [comment]
I'm still seeing this.
```PreHeat (missing)```

Which one should I install?
![image](https://user-images.githubusercontent.com/1277920/102721679-1c991e80-42db-11eb-9783-4d2651c21c0d.png)

**Comment by OllisGit** [comment]
Hi @leoheck ,

choose "Preheat Button".

I will update the Dialog in the next release to make it more clear.

PS. Sorry, for being late to the party, but I made a "Octoprint sabatical" (no coding, no printing) the last months.

**Comment by leoheck** [comment]
@OllisGit No problem, thanks for the feedback and the help. This is something needed for new users like me.

**Comment by OllisGit** [comment]
fyi: Currently I have some "private challenges to solve", thats the reason why It takes so long to release a new version, sorry!

Today I released a new development version (1.11.0). This version fix the issue. Please test and give feedback.

Thx, in advance
Olli
